operator/config/crd/kustomization.yaml

```diff
@@ -2,37 +2,35 @@
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
 resources:
-  - bases/execution.securecodebox.io_scans.yaml
-  - bases/execution.securecodebox.io_scantypes.yaml
-  - bases/execution.securecodebox.io_scancompletionhooks.yaml
-  - bases/execution.securecodebox.io_parsedefinitions.yaml
-  - bases/execution.securecodebox.io_scheduledscans.yaml
-  - bases/cascading.securecodebox.io_cascadingrules.yaml
-# +kubebuilder:scaffold:crdkustomizeresource
+- bases/execution.securecodebox.io_scans.yaml
+- bases/execution.securecodebox.io_scantypes.yaml
+- bases/execution.securecodebox.io_parsedefinitions.yaml
+- bases/execution.securecodebox.io_scancompletionhooks.yaml
+- bases/cascading.securecodebox.io_cascadingrules.yaml
+- bases/execution.securecodebox.io_scheduledscans.yaml
+#+kubebuilder:scaffold:crdkustomizeresource
```

Comment:
Should there not be an indent?

Reply:
Doesn't need to be.
Would prefer not to change it as this file was auto generated and might be unindented again on the next generation run.